loadtest/user/user.go

```
@@ -36,4 +36,7 @@ type User interface {
 	GetChannelStats(channelId string) error
 
 	// teams
+
+	// users
+	GetUsersStatusesByIds(userIds []string) ([]*model.Status, error)
```

Let's just return an error for now.

---

Done